src/main/java/net/glowstone/entity/passive/GlowOcelot.java

```diff
@@ -1,15 +1,20 @@
 package net.glowstone.entity.passive;
 
+import com.google.common.collect.Sets;
+import java.util.Set;
 import lombok.Getter;
 import net.glowstone.entity.meta.MetadataIndex;
 import org.bukkit.Location;
+import org.bukkit.Material;
 import org.bukkit.Sound;
 import org.bukkit.entity.AnimalTamer;
 import org.bukkit.entity.EntityType;
 import org.bukkit.entity.Ocelot;
 
 public class GlowOcelot extends GlowTameable implements Ocelot {
 
+    private static final Set<Material> BREEDING_FOODS = Sets.immutableEnumSet(Material.RAW_FISH);
+
```

Comment:
Doesn't raw salmon also work?

Reply:
Before 1.13, raw salmon is not a specific value of Material, it just has the data byte set to 1 so this is going to work on 1.12.*. However, it will need to be updated for 1.13.